index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/electron/update-electron-app
 // Definitions by: HashimotoYT <hashimoto.stream@gmail.com>
 
-export function updater(
+export default function updater(
```

Comment:
```suggestion
exports = function updater(
```

Reply:
oops. Thanks for taking a closer look, @MarshallOfSound